pkg/algorithm/cstorpoolselect/v1alpha1/select_test.go

```diff
@@ -1,8 +1,16 @@
 package v1alpha1
 
 import (
+	"errors"
 	"reflect"
 	"testing"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	cstorPoolUIDLabelKey string = "cstorpool.openebs.io/uid"
```

Comment:
It will be preferable to have this in source file.
```go
type labelKey string

const (
  cstorPoolUIDLabelKey labelKey = "cstorpool.openebs.io/uid"
)
```

Reply:
done